[2025-03-14T13:57:52.801+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T13:57:52.815+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mtrix_vendas_edi.process_txt_to_postgres manual__2025-03-14T13:57:51.206365+00:00 [queued]>
[2025-03-14T13:57:52.822+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mtrix_vendas_edi.process_txt_to_postgres manual__2025-03-14T13:57:51.206365+00:00 [queued]>
[2025-03-14T13:57:52.822+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T13:57:52.830+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_txt_to_postgres> on 2025-03-14 13:57:51.206365+00:00
[2025-03-14T13:57:52.835+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=72) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T13:57:52.836+0000] {standard_task_runner.py:72} INFO - Started process 74 to run task
[2025-03-14T13:57:52.837+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mtrix_vendas_edi', 'process_txt_to_postgres', 'manual__2025-03-14T13:57:51.206365+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/mtrix_vendas_edi.py', '--cfg-path', '/tmp/tmpoj7a8lny']
[2025-03-14T13:57:52.839+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask process_txt_to_postgres
[2025-03-14T13:57:53.010+0000] {task_command.py:467} INFO - Running <TaskInstance: mtrix_vendas_edi.process_txt_to_postgres manual__2025-03-14T13:57:51.206365+00:00 [running]> on host 266a86bf832d
[2025-03-14T13:57:53.070+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='user' AIRFLOW_CTX_DAG_ID='mtrix_vendas_edi' AIRFLOW_CTX_TASK_ID='process_txt_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T13:57:51.206365+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-14T13:57:51.206365+00:00'
[2025-03-14T13:57:53.071+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T13:57:53.089+0000] {logging_mixin.py:190} INFO - Nenhum arquivo VENDAS*.txt encontrado em: /opt/***/data
[2025-03-14T13:57:53.089+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T13:57:53.096+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T13:57:53.097+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=mtrix_vendas_edi, task_id=process_txt_to_postgres, run_id=manual__2025-03-14T13:57:51.206365+00:00, execution_date=20250314T135751, start_date=20250314T135752, end_date=20250314T135753
[2025-03-14T13:57:53.130+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T13:57:53.146+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-14T13:57:53.147+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
